packages/integration-karma/test/light-dom/slotting/index.spec.js

```diff
@@ -53,3 +54,14 @@ describe('imperative slotted content', () => {
         });
     });
 });
+
+describe('in shadow dom', () => {
+    it('appending to host should not throw error', () => {
+        expect(() => {
+            const element = createElement('x-container', { is: Shadow });
+            document.body.appendChild(element);
+        }).not.toThrowError(
+            /Invalid DOM operation on <x-container>\. Light DOM component don't allow imperative slotted content\./
+        );
```

Comment:
This test is currently too specific. This test will keep passing if the engine starts throwing an error with a different message than the current one.

```suggestion
            const element = createElement('x-container', { is: Shadow });
            document.body.appendChild(element);
```

I would also recommend updating the test to make sure the `div` element is actually a child a `x-container`.